lib/shared/widgets/custom_navigation_drawer.dart

```diff
@@ -119,13 +118,7 @@ class CustomNavigationDrawer extends StatelessWidget {
                     context,
                     checkAuthRequest: CheckAuthRequest(
                       navigate: (context, {isUSUser}) async {
-                        if (CachedFamilyUtils.isFamilyCacheExist()) {
-                          context.goNamed(
-                            FamilyPages.cachedChildrenOverview.name,
-                          );
-                        } else {
-                          context.goNamed(FamilyPages.childrenOverview.name);
-                        }
+                        context.goNamed(FamilyPages.childrenOverview.name);
                         unawaited(
                           AnalyticsHelper.logEvent(
```

Comment:
Nice, but this reminds me we dont use this widget in US anymore, maybe we need a ticket to clean up all 'isUSUser' code bc, we no longer have any shared widgets/pages with EU

Reply:
Good one, will make a ticket for it.

Reply:
https://linear.app/givt/issue/KIDS-1504/clean-up-us-widget-references-in-eu-code